src/idl_gen_json_schema.cpp

```diff
@@ -203,31 +217,47 @@ class JsonSchemaGenerator : public BaseGenerator {
           }
         }
         required_string.append("],");
-        code_ += required_string;
+        code_ += required_string + NewLine();
       }
-      code_ += "      \"additionalProperties\" : false";
-      std::string closeType("    }");
+      code_ += Indent(3) + "\"additionalProperties\" : false" + NewLine();
+      std::string closeType(Indent(2) + "}");
       if (*s != parser_.structs_.vec.back()) { closeType.append(","); }
-      code_ += closeType;  // close type
+      code_ += closeType + NewLine();  // close type
     }
-    code_ += "  },";  // close definitions
+    code_ += Indent(1) + "}," + NewLine();  // close definitions
 
     // mark root type
-    code_ += "  \"$ref\" : \"#/definitions/" +
-             GenFullName(parser_.root_struct_def_) + "\"";
+    code_ += Indent(1) + "\"$ref\" : \"#/definitions/" +
+             GenFullName(parser_.root_struct_def_) + "\"" + NewLine();
+
+    code_ += "}" + NewLine();  // close schema root
+    return true;
+  }
 
-    code_ += "}";  // close schema root
+  bool save() {
     const std::string file_path =
         GeneratedFileName(path_, file_name_, parser_.opts);
-    const std::string final_code = code_.ToString();
-    return SaveFile(file_path.c_str(), final_code, false);
+    return SaveFile(file_path.c_str(), code_, false);
+  }
+
+  const std::string getJson()
```

Comment:
use style consistent with the rest of the code, `get_json` and `{` on the same line.

Reply:
Done.